PGPAuth/src/main/java/org/lf_net/pgpunlocker/Server.java

```diff
@@ -12,15 +12,19 @@
     String _name;
     String _url;
     String _apgKey;
+    String _openpgpgKey;
```

Comment:
pgpg? Looks like a nice combination of PGP and GPG, but please only use one ^^
I would prefer PGP

Reply:
Can you reuse apgKey? (did we have that question before :O )